### PR TITLE
[UI] remove relative import and fix log level

### PR DESCRIPTION
**Pull request by KepingYan**
closes https://github.com/intel/llm-on-ray/issues/72, closes https://github.com/intel/llm-on-ray/issues/57

### CODE REVIEW

File: docs/web_ui.md [hunk 1]
```diff
@@ -2,8 +2,14 @@
 
 LLM-on-Ray introduces a Web UI, allowing users to easily finetune and deploy LLMs through a user-friendly interface. Additionally, the UI includes a chatbot application, enabling users to immediately test and refine the models.
 
-## Setup
-Please follow [setup.md](setup.md) to setup the environment first.
+## Setup Base Environment
+Please follow [setup.md](setup.md) to setup the base environment first.
+
+## Setup UI Environment
+After activating the environment installed in previous step, please run the following script to install environment for ui.
```
Comment (xwu99) [inline docs/web_ui.md:9]:
```suggestion
After activating the environment installed in the previous step, please run the following script to install environment for ui.
```

Comment (xwu99) [inline docs/web_ui.md:9]:
```suggestion
After activating the environment installed in previous step, please run the following script to install environment for Web UI.
```

File: docs/web_ui.md [hunk 2]
```diff
@@ -2,8 +2,14 @@
...
+```bash
+$ dev/scripts/install-ui.sh
+```
 
 ## Start Web UI
```
Comment (xwu99) [inline docs/web_ui.md:15]: Need to rephrase: 
# Get urls from the log
# Running on local URL:  http://0.0.0.0:8080
# Running on public URL: https://180cd5f7c31a1cfd3c.gradio.live

Get URL from the command line output (E.g. `http://0.0.0.0:8080` for local network and `https://180cd5f7c31a1cfd3c.gradio.live` for public network) and use the browser to open it.

File: ui/start_ui.py [hunk 3]
```diff
@@ -19,20 +19,23 @@
...
 
-sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
-from inference.inference_config import all_models, ModelDescription, Prompt
-from inference.inference_config import InferenceConfig as FinetunedConfig
-from inference.chat_process import ChatModelGptJ, ChatModelLLama  # noqa: F401
-from inference.predictor_deployment import PredictorDeployment
+ui_folder = os.path.dirname(__file__)
+sys.path.append(os.path.join(ui_folder, ".."))
+sys.path.append(os.path.join(ui_folder, "../inference"))
+from predictor_deployment import PredictorDeployment
```
Comment (xwu99) [inline ui/start_ui.py:25]: insert a blank line

Comment (xwu99) [inline ui/start_ui.py:25]: insert a blank line

File: ui/start_ui.py [hunk 4]
```diff
@@ -837,7 +830,7 @@ def _init_ui(self):
                     base_models_list.append("specify other models")
                     base_model_dropdown = gr.Dropdown(
                         base_models_list,
-                        value=base_models_list[2],
+                        value=base_models_list[4],
```
Comment (xwu99) [inline ui/start_ui.py:833]: add comment for using constant, what is the value you are specifying here. Same below

**Review by xwu99 — CHANGES_REQUESTED** [review]

**Comment by KepingYan** [comment]
Close this first as I‘ll address https://github.com/intel/llm-on-ray/issues/72 and https://github.com/intel/llm-on-ray/pull/90 in two separate PRs. Thanks @xwu99.

**Pull request closed without merging**